lib/dfe/analytics/services/entity_table_checks.rb

```diff
@@ -78,6 +78,14 @@ def determine_order_column(entity_name, columns)
           end
         end
 
+        def null_values_in_column?(column)
+          connection.select_value(<<-SQL).to_i.positive?
+            SELECT COUNT(*)
```

Comment:
@ericaporter maybe check whether being specific as to any field name instead of using * in the COUNT() is more efficient? Certainly in column stores avoiding COUNT(*) is a huge efficiency boost but this is running in Postgres which is a row store.

e.g. ```SELECT COUNT(#{column})```...

Reply:
@stevenleggdfe I opted for COUNT(*) because I thought COUNT{column} would ignore null values...

Reply:
So it does! That is one scary difference between PostgreSQL and GoogleSQL. Ignore me :-)

Reply:
@asatwal @stevenleggdfe we do another COUNT(*) in the main checksum calculation and I considered having both as count(id) but I couldn't work out of there was a possibility we might have null id values that we should be including. If the performance benefits would outweigh this likelihood, I'm happy to update, but as this fix is about not excluding nulls, I felt it was tempting fate a little 😄